base/docs/basedocs.jl

```diff
@@ -1557,12 +1557,22 @@ julia> false * NaN
 """
 Bool
 
+# IEEE 754 standard
+binary = Dict(
+    16 => Dict(:sign => 1, :exp => 5, :frac => 10),
+    32 => Dict(:sign => 1, :exp => 8, :frac => 23),
+    64 => Dict(:sign => 1, :exp => 11, :frac => 52),
+)
+
 for bit in (16, 32, 64)
```

Comment:
```suggestion
for (bit, sign, exp, frac) in ((16, 1, 5, 10), ...
```

Reply:
Just to expand on Jeff's comment, I think he's suggesting that it would be better to put these values inline in the for loop's RHS instead of making a Dict out of them.

Reply:
Yeah, that's nicer.